pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -101,6 +102,39 @@ def syncD2H(self, use_async=False):
         self._kokkos_state.DeviceToHost(self._state.ravel(order="C"))
         self._pre_rotated_state = self._state
 
+    def print_configuration(self, keyname=None):
```

Comment:
Rather than printing the value, we should be able to simply return a formatted datastructure. Printing still prevents access to the string information.

Also, since the device config is independent of the instance, we should probably create it as a static method. Since we should only need to call it once, we could also define a class-variable with the output dictionary, and reuse it throughout.

Reply:
Great suggestion! Done. Added a classmethod to return it configuration info dict and a class variable `kokkos_config`.